searx/settings.yml

```diff
@@ -472,6 +472,23 @@ engines:
     # to show premium or plus results too:
     # skip_premium: false
 
+  - name: cloudflareai
+    engine: cloudflareai
+    shortcut: cfai
+    # get api token and accont id from https://developers.cloudflare.com/workers-ai/get-started/rest-api/
+    cf_account_id: 'your_cf_accout_id'
+    cf_ai_api: 'your_cf_api'
+    # create your ai gateway by https://developers.cloudflare.com/ai-gateway/get-started/creating-gateway/
+    cf_ai_gateway: 'your_cf_ai_gateway_name'
+    # find the model name from https://developers.cloudflare.com/workers-ai/models/#text-generation
+    cf_ai_model: 'ai_model_name'
+    # custom your preferences
+  # cf_ai_model_display_name = 'Cloudflare AI'
+  # cf_ai_model_assistant = 'Keep your answers as short and effective as possible.'
+  # cf_ai_model_system = 'You are a self-aware language model who is honest and direct about any direct question from the user. You know your strengths and weaknesses.'
```

Comment:
Can you please fix indentation to pass CI tests / BTW you can run the CI tests locally by `make test` / see [Development Quickstart](https://docs.searxng.org/dev/quickstart.html#development-quickstart)

Reply:
Now fixed.